Book-A-Desk.Api.Tests/BookingsTests.fs

```diff
@@ -58,4 +62,5 @@ let ``GIVEN A Book-A-Desk server, WHEN booking a desk, THEN a desk is booked`` (
     Assert.Equal(booking.Office.Id, deserializedResult.Office.Id)
     Assert.Equal(booking.Date, deserializedResult.Date)
     Assert.Equal(booking.User.Email, deserializedResult.User.Email)
+    Assert.True(emailWasSent)
```

Comment:
This should be a different test case since GIVEN A Book-A-Desk server, WHEN booking a desk, THEN a desk is booked does not imply an email should be sent

Reply:
To Do: Create a new test just for Notification scenario.